templates/Template.NEP17.CSharp/NEP17.Methods.cs

```diff
@@ -30,7 +29,22 @@ public static bool Transfer(UInt160 from, UInt160 to, BigInteger amount)
             AssetStorage.Increase(to, amount);
 
             OnTransfer(from, to, amount);
+
+            // Validate payable
+            if (IsContract(to)) Contract.Call(to, "onPayment", new object[] { from, amount, data });
             return true;
         }
+
+        public static void OnPayment(UInt160 from, BigInteger amount, object data)
+        {
+            if (AssetStorage.GetPaymentStatus())
+            {
+                Payment(from, amount, data);
```

Comment:
```suggestion
                Payment(from, amount);
```
https://github.com/neo-project/proposals/pull/126#discussion_r530927491

Reply:
ahh you are right xD it was the event